templates/admin.tpl.php

```diff
@@ -0,0 +1,127 @@
+<?php
+$module_path = backdrop_get_path('module', 'tawk_to');
+backdrop_add_css($module_path . '/css/tawk_to.admin.css', 'file');
+?>
+
+<div id="content" >
+  <div class="l-wrapper">
+    <div class="l-wrapper-inner container container-fluid">
+      <div class="l-messages success-message-container" role="status" aria-label="Status messages" id="optionsSuccessMessage">
+        <div class="messages status">
+          <h2 class="element-invisible">Status message</h2>
+          Successfully set widget options to your site
+        </div>
+      </div>
+      <div class="l-page-title">
+        <a id="main-content"></a>
+      </div>
+      <div class="l-content" role="main" aria-label="Main content">
+        <div>
+          <fieldset class="form-wrapper" id="property-widget-settings">
+            <legend><span class="fieldset-legend">Property and Widget Selection</span></legend>
+            <div class="fieldset-wrapper">
+              <div class="fieldset-description">Select the property and widget from your tawk.to account</div>
+              <?php if (!$sameUser) { ?>
+                <div id="widget_already_set" class="alert alert-warning">Notice: Widget already set by other user</div>
+              <?php } ?>
+              <iframe id="tawkIframe" class="tawkto-property-widget-selection" src="" ></iframe>
+              <input type="hidden" class="hidden" name="page_id" value="<?php echo $widget['page_id']?>">
+              <input type="hidden" class="hidden" name="widget_id" value="<?php echo $widget['widget_id']?>">
+            </div>
+          </fieldset>
+		  <?php print $visibility_opts_form ?>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<script type="text/javascript">
```

Comment:
this javascript should be included in a separate `.js` file, and added to the page in the same way the CSS was added, using `backdrop_add_js()`.